com.unity.perception/Runtime/GroundTruth/Labelers/SemanticSegmentationLabeler.cs

```diff
@@ -167,6 +177,9 @@ protected override void Setup()
 
             m_SemanticSegmentationTextureReader = new RenderTextureReader<Color32>(targetTexture, myCamera,
                 (frameCount, data, tex) => OnSemanticSegmentationImageRead(frameCount, data));
+        
+            supportsVisualization = true;
+            EnableVisualization(supportsVisualization);
```

Comment:
Should this be called automatically?

Reply:
This kind of got cleaned up by turning it into a property. Now it appears as `visualizationEnabled = supportsVisualization`.  So it is getting called automatically, but if the user does not want their labeler to visualize, they could set supportsVisualization to false